yarn.lock

```diff
@@ -2,6 +2,41 @@
 # yarn lockfile v1
 
 
+"@ant-design/colors@^6.0.0":
+  version "6.0.0"
+  resolved "https://registry.yarnpkg.com/@ant-design/colors/-/colors-6.0.0.tgz#9b9366257cffcc47db42b9d0203bb592c13c0298"
+  integrity sha512-qAZRvPzfdWHtfameEGP2Qvuf838NhergR35o+EuVyB5XvSA98xod5r4utvi4TJ3ywmevm290g9nsCG5MryrdWQ==
+  dependencies:
+    "@ctrl/tinycolor" "^3.4.0"
```

Comment:
@kelvin-muchiri must these be installed?

Reply:
@moshthepitt Sure, I had added `antd` as a peer dependency, but for seems like at some point I might have moved it again as dependency....

Reply:
@moshthepitt We had a discussion with @p-netm & @KipSigei to resolve this, we are faced with the following quick options:

1. Reveal to allow this since Antd is used by `@opensrp/form-config` and some of its opensrp dependencies
2. Move Antd in `@opensrp/config` and its opensrp dependencies into peer depencies

For options that will take time:

1. Move shared utilities into a new package `@opensrp/form-config-core` and have separate packages for the bootstrap and Antd components
2. Use one package with either bootstrap or Antd components and modify the look in the imported project through css

Reply:
Wasn't the initial intention for this to work without polluting Reveal with antd?

> Move shared utilities into a new package @opensrp/form-config-core and have separate packages for the bootstrap and Antd components

I was actually under the impression that this is what had been done all along especially because we knew form config was going to be used in Reveal sooner or later.

> Move Antd in @opensrp/config into peer dependencies. Do this too for opensrp packages installed by @opensrp/form-config particularly @opensrp/notifications

What's the reason that these packages do not already have antd as a peer dependency?

Reply:
> Wasn't the initial intention for this to work without polluting Reveal with and?

@moshthepitt   Reveal components were not polluted...I added Antd components in a new directory but moved shared code into a utilities file within the same package

Reply:
> Reveal components were not polluted...I added Antd components in a new directory but moved shared code into a utilities file within the same package

I meant polluted in the sense that every Reveal user would have to pay the cost of downloading antd when interacting with the web app for no (obvious or not) benefit.

Reply:
> I was actually under the impression that this is what had been done all along especially because we knew form config was going to be used in Reveal sooner or later.

Moving Antd to peer dependencies seemed straight forward. But other opensrp packages have it installed as a dependency, in this case `@opensrp/notifications` installed in `@opensrp/form-config`. Therefore every opensrp package should have Antd as a peer dependency and not as a dependency?

Reply:
I don't have an answer for that, but I would say:

> Therefore, should every opensrp package have Antd as a peer dependency and not as a dependency?

1. Is there a reason why not?
2. Doing re-usable components stop making sense if using just one such component means pulling down a huge dependency tree.  If we have fully committed to reusable packages then this is something we must keep in mind.

Reply:
@moshthepitt Ok...Then for now looks like we shall take 

> Move Antd in @opensrp/config into peer dependencies. Do this too for opensrp packages installed by @opensrp/form-config particularly @opensrp/notifications

Reply:
Cool.  Let's pass it by @p-netm and @KipSigei for a sanity check.

Reply:
@moshthepitt This will not work either since the components using Antd are failing.....Seems like the next option is 
> Move shared utilities into a new package @opensrp/form-config-core and have separate packages for the bootstrap and Antd components

Reply:
@kelvin-muchiri what's an ETA for that?  I want to loop in @samkanga

Reply:
@moshthepitt  I'd give it 1 day